### DISCUSSION TITLE
(10) smart2go [P.20170606]

### DISCUSSION THREAD

**Discussion by progettistamatto** [sapienzastudentsnetwork/basi-di-dati-2#28] [Progetti]
In questa discussion a tema il progetto [smart2go](https://elearning.uniroma1.it/pluginfile.php/1350504/mod_folder/content/0/%285%29%20P%20-%20Progetti/%2810%29%20smart2go%20-%20BD2.pdf), discuteremo di eventuali proposte di soluzione, dubbi o scelte progettuali in merito alla realizzazione del progetto in questione

**Comment by daromacarlo** [comment]
https://github.com/daromacarlo/Triennale-in-Informatica/tree/main/Basi%20di%20dati/Basi%20di%20dati%20MODULO%202/Progetti/10.Smart2go
